integration-test/2022-disputed-borders.py

```diff
@@ -126,3 +127,251 @@ def test_admin_level_3_other_place(self):
+                'ne_id': '1746705405',
+                'recognized_by': 'RU',
+                'type': 'linestring',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 202058477,
+                'kind': 'disputed_claim',
+                'kind:cn': 'country',
+                'kind:tw': 'country',
+                'kind:in': 'unrecognized_country',
+                'kind:pk': 'unrecognized_country',
+                'kind:tr': 'unrecognized_country',
+                'kind:ru': 'country',
+            })
+
+    def test_boundary_claim_disputed_by_only(self):
+        z, x, y = (16, 53533, 28559)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/13574166
+            dsl.way(202058477, dsl.tile_diagonal(z, x, y), {
+                'admin_level': '2',
+                'boundary': 'claim',
+                'disputed_by': 'IN;PK;TR',
+                'name': 'Extent of Chinese Claim - ﻿Bara Hoti area',
+                'name:de': 'Ausdehnung des chinesischen Anspruchs',
+                'name:en': 'Extent of Chinese Claim - ﻿Bara Hoti area',
+                'name:zh': '中国声称边境线',
+                'name:zh_pinyin': 'Zhōngguó shēngchēng biānjìng xiàn',
+                'ne:brk_a3': 'B02',
+                'ne_id': '1746705405',
+                'type': 'linestring',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 202058477,
+                'kind': 'disputed_claim',
+                'kind:in': 'unrecognized_country',
+                'kind:pk': 'unrecognized_country',
+                'kind:tr': 'unrecognized_country',
+            })
+
+    def test_boundary_dispute_disputed_by_claimed_by(self):
+        z, x, y = (16, 53533, 28559)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/202058477
+            dsl.way(202058477, dsl.tile_diagonal(z, x, y), {
+                'admin_level': '2',
+                'boundary': 'disputed',
+                'claimed_by': 'CN;TW',
+                'disputed_by': 'IN',
+                'name': 'Extent', 'of': 'Chinese', 'Claim': 'at', 'Aksai': 'Chin',
+                'name:ur': 'اکسائی', 'چن': 'میں', 'چینی': 'دعوے', 'کی': 'حد',
+                'ne:brk_a3': 'B07',
+                'ne_id': '1746705319',
+                'recognized_by': 'RU;PK;TR',
+                'type': 'linestring',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'boundaries', {
+                'id': 202058477,
+                'kind:in': 'unrecognized_country',
+                'kind:cn': 'country',
+                'kind:tw': 'country',
+                'kind:ru': 'country',
+                'kind:pk': 'country',
+                'kind:tr': 'country',
+                'kind': 'disputed_reference_line',
+                'kind_detail': '2',
+            })
```

Comment:
Please also add test for the `disputed_id`, which should be a concat of `B07_1746705319`

Reply:
fixed in 2270fee074eae4f0940051964c1b3838a36efef9